modules/ovis/src/ovis.cpp

```diff
@@ -576,6 +578,57 @@ class WindowSceneImpl : public WindowScene
         node.setPosition(t);
     }
 
+    void getEntityAnimations(const String& name, std::vector<String>& out) CV_OVERRIDE
+    {
+        SceneNode& node = _getSceneNode(sceneMgr, name);
+        Entity* ent = dynamic_cast<Entity*>(node.getAttachedObject(name));
+        CV_Assert(ent && "invalid entity");
+        for (auto const& anim : ent->getAllAnimationStates()->getAnimationStates())
+        {
+            out.push_back(anim.first);
+        }
+    }
+
+    void playEntityAnimation(const String& name, const String& animname, bool loop = true) CV_OVERRIDE
+    {
+        SceneNode& node = _getSceneNode(sceneMgr, name);
+        Entity* ent = dynamic_cast<Entity*>(node.getAttachedObject(name));
+        CV_Assert(ent && "invalid entity");
+        AnimationState* animstate = ent->getAnimationState(animname);
+
+        if (ent->getSkeleton()->getBlendMode() != ANIMBLEND_CUMULATIVE)
```

Comment:
I think we should respect what is specified in the .skelton file instead of randomly overwriting it.

Reply:
I see that. It's not completely random though, see [here](https://github.com/OGRECave/ogre/blob/6089c69af600a92288b077e738968a9850fe8279/Samples/Character/include/SinbadCharacterController.h#L192).
The blend mode defaults to 'AVERAGE' if not serialized in the skeleton file, as it's the case for the __Ogre__ supplied meshes (the animations look terrible if the mode isn't properly set).

I'm adding the new _EntityProperty_ __ENTITY_ANIMBLEND_MODE__ and a corresponding `setEntityProperty` function override so that the user is able to override/set the blend mode accordingly (successfully tested).